cpp/refactor_module/algorithm/refactor.cpp

```diff
@@ -620,3 +620,31 @@ void Refactor::ExtractNode(mgp_list *args, mgp_graph *memgraph_graph, mgp_result
     return;
   }
 }
+
+void Refactor::RenameType(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+  mgp::MemoryDispatcherGuard guard{memory};
+  const auto arguments = mgp::List(args);
+  const auto record_factory = mgp::RecordFactory(result);
+  try {
+    const auto old_type{arguments[0].ValueString()};
+    const auto new_type{arguments[1].ValueString()};
+    const auto relationships{arguments[2].ValueList()};
+    auto graph{mgp::Graph(memgraph_graph)};
+
+    int64_t rels_changed{0};
+    for (const auto &relationship_value : relationships) {
```

Comment:
You are using here `const auto`, and line below you are not using `const`, I think it is bit misleading because by reading code you expect `relationship_value` not to change, but it changes, so I would remove `const`